### DISCUSSION TITLE
help! please!

### DISCUSSION THREAD

**Discussion by Vegabomb69** [community/community#123564] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

can someone help me get all this crap that my ex put on my phone? I have absolutely no idea how!

**Comment by AmimiHamza** [comment]
what's the problem ?

**Reply by Vegabomb69** [reply]
idk if any of that will help I'm absolutely clueless about this stuff I
just know it's happening and I'm so sick of this shit.

**Comment by Vegabomb69** [comment]
idk if any of that will help I'm absolutely clueless about this stuff I
just know it's happening and I'm so sick of this shit.

**Comment by Vegabomb69** [comment]
he admitted to downloading something from GitHub onto my phone or
"attempting" too but no matter what I can't get him off.

**Comment by ebndev** [comment]
Hi @Vegabomb69, The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
